src/compiler/emitter.ts

```diff
@@ -476,7 +481,9 @@ namespace ts {
 
         function emitMappedTypeParameter(node: TypeParameterDeclaration): void {
             emit(node.name);
-            write(" in ");
+            writeSpace(" ");

```

Comment:
why the argument? what other kinds of spaces are there?

Reply:
The distinction is for the symbol writer-style-usage. AFAIK, you're right and `" "` is the only whitespace we ever call it with. I'll push the literal to the edge where we interact with the writer.